sdks/java/io/xml/src/test/java/org/apache/beam/sdk/io/xml/XmlSourceTest.java

```diff
@@ -526,13 +560,21 @@ public void testReadXMLInvalidRecordClass() throws IOException {
     File file = tempFolder.newFile("trainXMLSmall");
     Files.write(file.toPath(), trainXML.getBytes(StandardCharsets.UTF_8));
```

Comment:
the pull request has lot my unit test, sorry. 
Take a look at it, and say me if you want something else